hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/AbstractYarnScheduler.java

```diff
@@ -1566,6 +1567,14 @@ public long checkAndGetApplicationLifetime(String queueName, long lifetime) {
     return lifetime;
   }
 
+  @Override
+  public CSQueue getOrCreateQueueFromPlacementContext(ApplicationId
```

Comment:
It shouldn't return a CSQueue, as Fair Scheduler or the Fifo Scheduler do not utilize Capacity Scheduler queues.